### ISSUE TITLE
User passwords are not hashed during updateAttribute as of 2.45

### ISSUE THREAD

**Issue by kblcuk** [strongloop/loopback-datasource-juggler#844]
Greetings!

For some reason, user password are not run through [password setter](https://github.com/strongloop/loopback/blob/27c9e263e0a1b003f8c0a7969119ea66f18c2bb4/common/models/user.js#L591-L602) in User model when updated via `updateAttribute`.

This seems to work as intended when using loopback-datasource-juggler 2.44

Versions of relevant (I think) libraries that I have at the moment:
```
npm list loopback loopback-connector-remote loopback-connector-mongodb loopback-datasource-juggler
├── loopback@2.26.2
├── loopback-connector-mongodb@1.13.3
├── loopback-connector-remote@1.0.3
└── loopback-datasource-juggler@2.45.0
```

Thanks!

**Comment by glesage** [comment]
Same issue as: https://github.com/strongloop/loopback/issues/2029

**Comment by glesage** [comment]
Ok def something in this commit: https://github.com/strongloop/loopback-datasource-juggler/commit/8ca67c3d48952b972bc2e2c1ee4c8c93b389f99c

**Comment by rawrmaan** [comment]
This broke our entire login system. It's **insane** that nothing has been done about this. Reverted to 2.44.0, thank you @kblcuk.

**Comment by bajtos** [comment]
@Amir-61 I think we have already fixed this issue, could you PTAL and link to the relevant PR/commit & juggler version that contains the fix?

**Comment by bajtos** [comment]
This may be a duplicate of https://github.com/strongloop/loopback/issues/2029

**Comment by Amir-61** [comment]
@bajtos Yes that's a duplicate of strongloop/loopback#2029 and it is fixed in  `loopback-datasource-juggler@2.45.1`. 

@kblcuk Please let me know if I can close this ticket?

Thanks!

**Comment by bajtos** [comment]
I'm closing this issue as a duplicate based on the comment above.
